### ISSUE TITLE
DeprecationWarnings in current version

### ISSUE THREAD

**Issue by graysky2** [MilhouseVH/texturecache.py#49]
Arch just went to python 3.7 and the script now throws some warnings 
```
% texturecache vclean
/usr/bin/texturecache:111: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = ConfigParser.SafeConfigParser(strict=False)
/usr/bin/texturecache:149: DeprecationWarning: This method will be removed in future versions.  Use 'parser.read_file()' instead.
  config.readfp(cfg)
Cleaning library...
```

**Comment by graysky2** [comment]
@MilhouseVH - Not sure what your email prefs are so tag :label:

**Comment by MilhouseVH** [comment]
Not sure when I'll have time to look at this, maybe later this week. If you have a patch that would be great. I'm not even sure if I have a test environment for this - presumably it should be possible to make the changes against Python 3.6.3 and remove the deprecated method.

**Comment by graysky2** [comment]
@MilhouseVH - I don't know python or else I would take a stab at a PR.  As to your test environment, what about an Arch Linux container?  Very easy to spin one up.

**Comment by graysky2** [comment]
@MilhouseVH - See PR#50 for a partial fix.  I am happy to update the PR if you have a suggestion to fix the other warning caused by line 149:
```
/usr/bin/texturecache:149: DeprecationWarning: This method will be removed in future versions.  Use 'parser.read_file()' instead.
  config.readfp(cfg)
```

**Comment by MilhouseVH** [comment]
Thanks for reporting this - sorry it took so long to get around to looking at this. Should be fixed now with v2.4.5.

**Comment by graysky2** [comment]
https://github.com/MilhouseVH/texturecache.py/releases/tag/2.4.5 works, thanks.
